code/modules/religion/sects/shadow_sect.dm

```diff
@@ -134,43 +146,61 @@
 		on_mob_leave(each_mob)
 	src.set_light(0, 0, DARKNESS_INVERSE_COLOR)
 
+/obj/structure/destructible/religion/shadow_obelisk/proc/var3_bucle_togle() // this is ussles untill it is inherited by var3
```

Comment:
I have absolutely no idea what the name of this proc means, it looks like random characters. Choose an appropriate name that explains what the proc is going to be doing.